plugins/serializers/prometheus/prometheus.go

```diff
@@ -32,10 +38,20 @@ const (
 	StringAsLabel
 )
 
+// ProtobufFormating defines how to format body request.
+type Formating string
+
+const (
+	FormatText        Formating = "text"
+	FormatRemoteWrite Formating = "remote_write"
+	Format
```

Comment:
blank intentional?

Reply:
No, I forgot to remove it, because before use boolean, I wrote code with string config.